src/Jellyfin.Plugin.Dlna/Eventing/DlnaEventManager.cs

```diff
@@ -18,24 +14,40 @@
 
 namespace Jellyfin.Plugin.Dlna.Eventing;
 
+/// <summary>
+/// Defines the <see cref="DlnaEventManager" />.
+/// </summary>
 public class DlnaEventManager : IDlnaEventManager
 {
     private readonly ConcurrentDictionary<string, EventSubscription> _subscriptions =
-        new ConcurrentDictionary<string, EventSubscription>(StringComparer.OrdinalIgnoreCase);
+        new(StringComparer.OrdinalIgnoreCase);
 
     private readonly ILogger _logger;
     private readonly IHttpClientFactory _httpClientFactory;
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="DlnaEventManager"/> class.
+    /// </summary>
+    /// <param name="logger">Instance of the <see cref="ILogger"/> interface.</param>
+    /// <param name="httpClientFactory">Instance of the <see cref="IHttpClientFactory"/> interface.</param>
     public DlnaEventManager(ILogger logger, IHttpClientFactory httpClientFactory)
     {
         _httpClientFactory = httpClientFactory;
         _logger = logger;
     }
 
-    public EventSubscriptionResponse RenewEventSubscription(string subscriptionId, string notificationType, string requestedTimeoutString, string callbackUrl)
+    /// <summary>
+    /// Renews an event subscription.
+    /// </summary>
+    /// <param name="subscriptionId">The subscription id.</param>
+    /// <param name="notificationType">The notification type.</param>
+    /// <param name="requestedTimeoutString">The requested timeout string.</param>
+    /// <param name="callbackUrl">The callback URL.</param>
+    /// <returns>EventSubscriptionResponse.</returns>
```

Comment:
```suggestion
    /// <returns>The response to the subscription request.</returns>
```
Or something else that isn't just the type.